src/components/CommentList/CommentList.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import CommentInfo from '../CommentInfo/index';
+
+type Props = {
+  comments: Comment[];
+};
+
+export const CommentList: React.FC<Props> = (props) => {
+  const { comments } = props;
+
+  return (
+    <div>
+      {comments.map(comment => (
+        <div key={comment.id}>
+          <div>
+            <CommentInfo {...comment} />
+          </div>
+        </div>
```

Comment:
Too many redundant div wrappers here. Also, as was mentioned before, don't use destructuring to pass your params.
```suggestion
        <CommentInfo key={comment.id} {...comment} />
```